memote/suite/cli/reports.py

```diff
@@ -217,19 +226,29 @@ def diff(models, filename, pytest_args, exclusive, skip, solver,
         config.merge(ReportConfiguration.load(custom))
     # Build the diff report specific data structure
     diff_results = dict()
+    loaded_models = list()
     for model_path in models:
         try:
             model_filename = os.path.basename(model_path)
             diff_results.setdefault(model_filename, dict())
             model = callbacks._load_model(model_path)
             model.solver = solver
-            _, diff_results[model_filename] = api.test_model(
-                model, results=True, pytest_args=pytest_args,
-                skip=skip, exclusive=exclusive, experimental=experimental)
+            loaded_models.append(model)
         except Exception as e:
```

Comment:
Rewrite this catch statement to:
```python
except (IOError, SBMLError):
    LOGGER.debug(exc_info=True)
    LOGGER.warning("An error occurred while loading the model '%s'.", model_filename)
```
And abort the process if `len(loaded_models) < 2`.

Reply:
Hopefully, all errors would be children of `IOError` but I'm not sure.